rpm/src/test/java/org/eclipse/packager/rpm/signature/RpmFileSignatureProcessorTest.java

```diff
@@ -7,45 +7,55 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  *
- * Contributors: 
+ * Contributors:
  *   mat1e, Groupe EDF - initial API and implementation
+ *   Jens Reimann, Red Hat, Inc
  ********************************************************************************/
 package org.eclipse.packager.rpm.signature;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.junit.jupiter.api.Assertions.fail;
 
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
 
-import org.bouncycastle.bcpg.ArmoredInputStream;
 import org.bouncycastle.openpgp.PGPException;
-import org.bouncycastle.openpgp.PGPPublicKey;
-import org.bouncycastle.openpgp.PGPPublicKeyRing;
-import org.bouncycastle.openpgp.bc.BcPGPPublicKeyRing;
+import org.eclipse.packager.rpm.HashAlgorithm;
 import org.eclipse.packager.rpm.RpmSignatureTag;
 import org.eclipse.packager.rpm.Rpms;
 import org.eclipse.packager.rpm.parse.InputHeader;
 import org.eclipse.packager.rpm.parse.RpmInputStream;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
 import org.junit.jupiter.api.Order;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestMethodOrder;
 
+import com.google.common.io.ByteStreams;
+
 @TestMethodOrder(OrderAnnotation.class)
 public class RpmFileSignatureProcessorTest {
 
     private static final String SOURCE_FILE_PATH = "src/test/resources/data/org.eclipse.scada-0.2.1-1.noarch.rpm";
+
     private static final String PRIVATE_KEY_PATH = "src/test/resources/key/private_key.txt";
+
     private static final String PUBLIC_KEY_PATH = "src/test/resources/key/public_key.txt";
-    private static final String RESULT_FILE_PATH = "src/test/resources/result/org.eclipse.scada-0.2.1-1.noarch.rpm";
-    private static final String RESULT_DIR = "src/test/resources/result";
+
+    private static final String RESULT_DIR = "target/test-data/signature";
```

Comment:
@ctron OK, some tests are writing directly to target/ which isn't really good and could possibly cause issues with repeated runs. I will create a PR to use JUnit 5 `@TempDir` for the output directory for any tests.